### PR TITLE
Updated doc for gpu specification

### PR DESCRIPTION
**Pull request by himanshu-kun**
**How to categorize this PR?**

/area documentation
/kind enhancement
/platform gcp

**What this PR does / why we need it**:
Changes made to deal with issue faced by customer https://sap-ti.slack.com/archives/CBV3JS9S4/p1693203852606049

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

**Review by kon-angelo — APPROVED** [review]
/lgtm

File: docs/usage/usage.md [hunk 1]
```diff
@@ -157,7 +157,10 @@ The worker configuration contains:
 
   **Note**: 
   * A rolling upgrade of the worker group would be triggered in case the `acceleratorType` or `count` is updated.
-  * Some machineTypes like [a2 family](https://cloud.google.com/blog/products/compute/announcing-google-cloud-a2-vm-family-based-on-nvidia-a100-gpu) come with already attached gpu of `a100` type and pre-defined count. If your workerPool consists of those machineTypes, please **do not** specify any GPU configuration.
+  * Some machineTypes like [a2 family](https://cloud.google.com/blog/products/compute/announcing-google-cloud-a2-vm-family-based-on-nvidia-a100-gpu) come with already attached gpu of `a100` type and pre-defined count. If your workerPool consists of such machineTypes, please specify exact GPU configuration for the machine type as specified in Google cloud documentation.  `acceleratorType` to use for families with attached gpu are stated below:
+    1) *a2 family* -> `nvidia-tesla-a100`
+    2) *g2 family* -> `nvidia-L4`
```
Comment (kon-angelo) [inline docs/usage/usage.md:162]: Are you sure about the `L4`? 
If I remember these are case-sensitive and the error message was mentioning `l4`. Can you doublecheck this @himanshu-kun  ?

**Review by kon-angelo — CHANGES_REQUESTED** [review]

File: docs/usage/usage.md (hunk 1 above)
Comment (himanshu-kun) [reply docs/usage/usage.md:162]: its lower-case 😬 .. thanks for catching it.

Comment (himanshu-kun) [reply docs/usage/usage.md]: updated. pls take a look

**Review by kon-angelo — APPROVED** [review]
/lgtm

**Pull request merged**